public/index.html

```diff
@@ -5,8 +5,25 @@
        <link href='https://bootswatch.com/yeti/bootstrap.min.css' rel='stylesheet' >
        <link href="/style.css" rel="stylesheet" type="text/css">
   </head>
-
   <body>
+    <script>
```

Comment:
Should just need to dispatch and use axios to hit `/api/auth/login/:strategy` (facebook)